### ISSUE TITLE
Attempts to Reproduce

### ISSUE THREAD

**Issue by leonnil** [ouyanghaodong/DEYO#13]
Hi, thanks for your great work!

I'm attempting to reproduce your model following the README file. After 96 epochs training of the model DEYO-tiny, I find its map (37.0%) is lower than the map you report in the paper (37.6%). I trained model in multi gpus setting with 8 * 3090, and the batch size is set to 128 (16 per GPU), while others are the same.

I can't figure out the problem. Could you please tell me your training details?

**Comment by ouyanghaodong** [comment]
Our models were all trained using a single GPU with a batch size of 16. The decrease in accuracy when using multiple GPUs may be due to an increase in your batch size. We have only trained DEYO-L on 8 GPUs, and its accuracy is identical to that when trained on a single GPU.

**Comment by ouyanghaodong** [comment]
I misunderstood your answer; I'm not sure if you've set `model.train(batch=128)`

**Comment by leonnil** [comment]
thx for your reply. I tried to train with a batch size of 16 (`model.train(batch=16, device = '0,1,2,3,4,5,6,7')`), and the training time has increased to nearly 35min/epoch. Is this normal?
In addition, how do you freeze the backbone and neck of YOLOv8, and where is the code located? I didn't see you passing the `freeze` parameter.

**Comment by ouyanghaodong** [comment]
When the CPU is the performance bottleneck, multiple GPUs may not have an acceleration effect. In some cases, it is not even as good as a single GPU. Using fewer GPUs may be faster. This is because hungarian-matching is more expensive. CPU resources, we also encountered this situatio。we did not add the backbone and neck parameters to the optimizer, and did not calculate the gradient of these parameters to implement freeze, but it should also be more concisely implemented through the freeze parameter of v8.
